web/embeddings_demo.html

```diff
@@ -54,7 +54,7 @@
             cupidatat non proident, sunt in culpa qui officia deserunt mollit anim
             id est laborum.
           </p>
-          <iframe src="embed-dart.html"></iframe>
+          <iframe src="embed-dart.html" sandbox="allow-scripts allow-forms allow-same-origin"></iframe>
```

Comment:
Does this sandboxing help? Ideally we wouldn't need to sandbox as we control the content we're embedding.